apps/expo/expo-plugins/with-modify-gradle.js

```diff
@@ -0,0 +1,23 @@
+//@ts-check
```

Comment:
add a comment here linking to our issue as well as the original issue on expo for future reference please

Reply:
Done 👍